app/routes/create.tsx

```diff
@@ -0,0 +1,100 @@
+
+import type { ActionFunction, ActionFunctionArgs, LoaderFunctionArgs } from "@remix-run/node";
+import { Form, redirect, useActionData } from "@remix-run/react";
+import { json } from "@remix-run/react";
+import { useLoaderData } from "@remix-run/react";
+import client from "~/client";
+
+import { prismaDatabase } from "~/prismaDatabase";
+
+
+export const action: ActionFunction = async ({ request }) => {
+  const data = await request.formData();
+
+  console.log("request.formData:", request.formData)
+
+  await client.survey.create({
+    data: {
+      name: data.get("surveyName") || "default survey name",
+    }
+  })
+
+  const surveysAfterAddition = await client.survey.findMany();
+
+  return redirect(`/survey/1`)
+
+  // const formData = await request.formData();
+  // const survey = await createSurvey(formData);
+
+  // const newSurveyId = ?!?
+
+  // return redirect(`/survey/${client.survey.newSurveyId}`)
+
+
+}
+
+
+
+export default function CreateSurveyForm () {
+  const actionData = useActionData<typeof action>();
+
+  return (
+    <div>
+      <Form method = "post" action="/create">
+  
+        <p>
+          <label>
+            Name: {" "}
+            <input 
+              name="surveyName" 
+              type="text"
+              defaultValue={actionData?.values.name} 
```

Comment:
`defaultValue` doesn't exist, and `actionData` has a different shape. I think you meant to use `<input placeholder="sometext">`.